velox/exec/GroupingSet.cpp

```diff
@@ -325,7 +325,7 @@ bool GroupingSet::getOutput(
   }
   for (int32_t i = 0; i < aggregates_.size(); ++i) {
     aggregates_[i]->finalize(groups, numGroups);
-    auto aggregateVector = result->childAt(i + totalKeys);
+    auto& aggregateVector = result->childAt(i + totalKeys);
```

Comment:
Another lost comment?

---
This looks like an unrelated performance improvement. The is a very subtle change and it is difficult to understand why It matters without an explanation. It would be cleaner to submit this in a separate PR and explain what was wrong / inefficient before and how this change makes things better.

Does this change reduce memory allocations? Is it possible to add a test to ensure this doesn't get accidentally broken in the future?